### ISSUE TITLE
Can't add "Text" sprite from library

### ISSUE THREAD

**Issue by Kenny2github** [LLK/scratch-gui#466]
### Expected Behavior

The "Text" sprite in the library should be added to the project when clicked

### Actual Behavior

When clicked, nothing happened

### Screenshot

![scratch-gui-text-sprite](https://user-images.githubusercontent.com/28599280/27424533-f39ad904-5767-11e7-83c3-70760bf3002d.gif)

### Steps to Reproduce

Press the + to add a new sprite, search "text", click the lone sprite that appears, and nothing will happen

### Operating System and Browser

Windows 10 Chrome 58

**Comment by paulkaplan** [comment]
@rschamp looks like one of the costume assets on this sprite does not exist, specifically 

```
                {
                    "costumeName": "text keep scratching",
                    "baseLayerID": -1,
                    "baseLayerMD5": "22a916248a294eadc07a7a065f5154e8.svg",
                    "bitmapResolution": 1,
                    "rotationCenterX": 135,
                    "rotationCenterY": 24
                },
```
which 404s on the URL https://cdn.assets.scratch.mit.edu/internalapi/asset/22a916248a294eadc07a7a065f5154e8.svg/get/

**Comment by christan1029** [comment]
Only the "keep scratching" costume URL 404's; the other costume URLs are fine. Any way to find the actual URL? I tried looking for it in the scratch-flash repo, but I wasn't able to find it.

This is the costume:
![image](https://user-images.githubusercontent.com/29719007/28345067-8dc02a4c-6bf5-11e7-876f-e424e8cf9b7f.png)

**Comment by liam4** [comment]
I've got the Scratch 2.0 link:

<pre>
<a href="https://cdn.assets.scratch.mit.edu/internalapi/asset/b3de9a77e2c1bb2f3e29f29a03d869ea.svg/get/">/asset/b3de9a77e2c1bb2f3e29f29a03d869ea.svg/get/</a> # Scratch 2.0
<a href="https://cdn.assets.scratch.mit.edu/internalapi/asset/22a916248a294eadc07a7a065f5154e8.svg/get/">/asset/22a916248a294eadc07a7a065f5154e8.svg/get/</a> # Scratch 3.0
</pre>

**Comment by liam4** [comment]
Interestingly, the sprite library's JSON is stored [in the scratch-gui codebase itself](https://github.com/LLK/scratch-gui/blob/4352783b18dd26e8ef0bb6ad71e6a62fd62714d3/src/lib/libraries/sprites.json).

..Whereas each sprite's JSON is stored in a separate file, e.g. [the "Text" sprite](https://cdn.assets.scratch.mit.edu/internalapi/asset/0151c613f210ee446f0a51967ca0c625.json/get/).

**Comment by christan1029** [comment]
So, could we just replace the broken 3.0 link with the 2.0 link?

**Comment by rschamp** [comment]
Fixed in #561
